authors/apps/articles/views.py

```diff
@@ -102,13 +85,27 @@ class UpdateDeleteArticleView(APIView):
     """
 
     # Route protection
-    permission_classes = (IsAuthenticated,)
+    permission_classes = (IsAuthenticatedOrReadOnly,)
 
     def get(cls, request, article_id):
         article = get_object_or_404(Article, id=article_id, delete_status=False)
         serializer = ArticleSerializer(article, many=False)
         an_article = dict(serializer.data)
         an_article['likeCount'] = [like_grand_count(an_article)]
+
+        current_user = request.user
+        # Make this functionality on available for logged in users.
+        if current_user and article.author.username != current_user.username:
```

Comment:
Great work @kalsmic, awesome logic, however, given the same functionality on [medium.com](https://medium.com/), also anonymous users are allowed a read/view count.

Reply:
@AtamaZack, I have implemented the feedback. now article views from an anonymous user add to the view stats of the article